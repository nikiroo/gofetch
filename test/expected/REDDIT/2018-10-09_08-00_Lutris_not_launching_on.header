0Lutris not launching on ubuntu	null/REDDIT/2018-10-09_08-00_Lutris_not_launching_on		70
i  [linux_gaming] 2018-10-09_08-00
i
