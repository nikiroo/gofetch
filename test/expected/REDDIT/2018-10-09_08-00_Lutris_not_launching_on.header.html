<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h2><a href='2018-10-09_08-00_Lutris_not_launching_on.html'>Lutris not launching on ubuntu</a></h2>
	<div class='details'>([linux_gaming] 2018-10-09_08-00)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		
	</div>
<hr/>
</div>
</body>
